src/client.rs

```diff
@@ -18,18 +19,26 @@ const MAX_ADDR_LEN: usize = 260;
 
 #[derive(Debug)]
 pub struct Config {
+    /// Timeout of the command request
+    request_timeout: Option<u64>,
```

Comment:
This is not a timeout related to request, but to the connection.

`connect_timeout` would be more preferable imo, what do you think?

Reply:
I do agree, but unless I misunderstood, it seemed to me the same for the server functionality?
I also wonder if we also need another timeout for sending content as well as receiving content. As that is currently not something we (can) configure either, right?

Reply:
Changed it for the client.
I do feel we ought to change it for the server as well,
as it is used for the exact same purpose. But that would be a breaking change, so didn't touch it yet.